xml/System.Security.Cryptography.X509Certificates/X509ChainPolicy.xml

```diff
@@ -186,7 +184,7 @@
         <ReturnType>System.Security.Cryptography.X509Certificates.X509Certificate2Collection</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Represents an additional collection of certificates that can be searched by the chaining engine when validating a certificate chain.</summary>
+        <summary>Gets an object that represents an additional collection of certificates that can be searched by the chaining engine when validating a certificate chain.</summary>
         <value>An <see cref="T:System.Security.Cryptography.X509Certificates.X509Certificate2Collection" /> object.</value>
         <remarks>
```

Comment:
nit: for the Remarks section, since the property is read-only and the collection is empty, how are items added to it?

Reply:
The collection cannot be replaced, but it's still mutable.  Add and Clear work on it just fine.